media_driver/linux/ult/ult_app/driver_loader.h

```diff
@@ -96,7 +96,7 @@ struct DriverSymbols
     bool Initialized() const
     {
         auto p = (const void * const *)this;
-        for (auto i = 0; i < sizeof(this) / sizeof(const void *); i++)
+        for (auto i = 0; i < sizeof(*this) / sizeof(const void *); i++)
```

Comment:
Folks, please, remove this and check pointers directly. This is simply a freak over complex implementation of very simple thing: check 5 pointers that they are not NULL. You don't need any loop here!!

Reply:
Yes, checking 5 pointers is simple and easy to understand than previous code.